ciscoasa.json

```diff
@@ -3,22 +3,20 @@
     "name": "Cisco ASA",
     "description": "This app supports containment actions in addition to investigative actions on a Cisco ASA device",
     "type": "firewall",
-    "publisher": "Splunk",
+    "publisher": "Splunk Community",
     "main_module": "ciscoasa_connector.py",
     "app_version": "2.0.4",
     "utctime_updated": "2022-01-07T20:56:17.000000Z",
     "package_name": "phantom_ciscoasa",
     "product_vendor": "Cisco Systems",
     "product_name": "Cisco ASA",
     "product_version_regex": ".*",
-    "min_phantom_version": "4.9.39220",
-    "latest_tested_versions": [
-        "On Prem, Version 7.6"
-    ],
+    "min_phantom_version": "5.1.0",
+    "fips_compliant": true,
```

Comment:
remove this or mark as a false https://splunk.slack.com/archives/GBSGHUUD9/p1643701225287839?thread_ts=1639635017.100200&cid=GBSGHUUD9